autopush/db.py

```diff
@@ -407,6 +421,12 @@ def store_message(self, uaid, channel_id, message_id, ttl, data=None,
                       headers=None, timestamp=None):
         """Stores a message in the message table for the given uaid/channel
         with the message id"""
+        if not uaid:
```

Comment:
yeah, the only problem is that we need to get the logic right, and I wanted to be very explicit about what's being checked. I've tried to do clever logic chains in the past and failed horribly. I just default to simpler and pay the minor opcode penalty.